styleguide/derek/view-templates/resource-center/_resource-center.scss

```diff
@@ -93,7 +93,7 @@
 }
 
 @include resourcecenter-type('.resourcecenter-pdf', $brand-primary, '\e8c5', 'pdf-template-graphic.png');
-@include resourcecenter-type('.resourcecenter-code', $purple, '\e8c4', 'code-template-graphic.png');
-@include resourcecenter-type('.resourcecenter-docs', $teal-dark, '\e808', 'suppoerDocs-template-graphic.png');
+@include resourcecenter-type('.resourcecenter-code', $purple, '\e808', 'code-template-graphic.png');
+@include resourcecenter-type('.resourcecenter-docs', $teal-dark, '\e847', 'suppoerDocs-template-graphic.png');
 @include resourcecenter-type('.resourcecenter-webinar', $mustard, '\e8c6', 'webinar-template-graphic.png');
```

Comment:
Would you mind also changing them here too? 
https://github.com/rackerlabs/zoolander/blob/master/styleguide/derek/pattern-components/resourceCenter-resource/_resourceCenter-resource.scss#L86
```sass
@include resource-iconBorderColor('.resource-codeIcon', $purple, '\e8c4');
@include resource-iconBorderColor('.resource-supportIcon', $teal-dark, '\e808');
@include resource-iconBorderColor('.resource-pdfIcon', $brand-primary, '\e8c5');
@include resource-iconBorderColor('.resource-videoIcon', $blue-gray, '\e8c8');
@include resource-iconBorderColor('.resource-webinarIcon', $mustard, '\e8c6');
```